### ISSUE TITLE
Accessibility finding: US flag image doesn't have description

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#139060]
### Checklist item

WEB-111-001 Meaningful descriptions are provided for informative images

### Severity level

3, Medium

### Page(s)/url(s)

This is an issue with va-minimal-header, and was discovered on this page:

https://staging.va.gov/my-va/submissions/9093f30e-0d09-43ff-a13f-087847063071?postLogin=true

### Details

Form viewer page - The US flag at the top of the page is a meaningful image, but is given the role of "presentation", which hides it from assistive technology.

<img width="710" height="73" alt="Image" src="https://github.com/user-attachments/assets/2d1c58d0-cffd-4f07-bc69-b63716fefbbc" />

### Reproduction steps

Inspect the code for the US flag at the top of the page. Go to the accessibility panel.

### Proposed solution or next steps

This should match the implementation across VA.gov - remove `role="presentation"` and add `alt="U.S. Flag"`.

**Comment by rileyorr** [comment]
The team reviewed this issue on April 15th and concluded this is a platform level issue

**Comment by rileyorr** [comment]
I'm going to unassign myself from this ticket; you can tag me directly or reach my team in #benefits-employee-experience with questions or updates on this work!
